Framework/PythonInterface/plugins/algorithms/PEARLTransfit.py

```diff
@@ -0,0 +1,370 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+# -----------------------------------------------------------------------
+# Transfit v2 - translated from Fortran77 to Python/Mantid
+# Original Author: Christopher J Ridley
+# Last updated: 2nd October 2020
+#
+# Program fits a Voigt function to PEARL transmission data, using the doppler broadening
+# of the gaussian component to determine the sample temperature
+#
+#
+# References:
+# 'Temperature measurement in a Paris-Edinburgh cell by neutron resonance spectroscopy' - Journal Of Applied Physics 98,
+# 064905 (2005)
+# 'Remote determination of sample temperature by neutron resonance spectroscopy' - Nuclear Instruments and Methods in
+# Physics Research A 547 (2005) 601-615
+# -----------------------------------------------------------------------
+from mantid.kernel import Direction
+from mantid.api import PythonAlgorithm, MultipleFileProperty, AlgorithmFactory, mtd
+from mantid.simpleapi import (LoadRaw, DeleteWorkspace, CropWorkspace, NormaliseByCurrent, ExtractSingleSpectrum,
+                              ConvertUnits, CreateWorkspace, Fit, Plus, Multiply, RenameWorkspace,
+                              CreateSingleValuedWorkspace)
+from os import path
+import numpy as np
+
+
+class PEARLTransfit(PythonAlgorithm):
+
+    def version(self):
+        return 1
+
+    def name(self):
+        return 'PEARLTransfit'
+
+    def category(self):
+        return 'PEARL tools'
```

Comment:
I think this has been flagged up by one of the Jenkins jobs because the category doesn't exist in a master list somewhere. I think rather than create a new category (which then shows up in the algorithm tree in the Mantid UI) it would be better to pick an existing category. I suggest you add it to the category "Diffraction\\Fitting" category